app/views/admin/document_collection_group_memberships/index.html.erb

```diff
@@ -0,0 +1,62 @@
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", {
+    href: admin_document_collection_groups_path(@collection),
+  } %>
+<% end %>
+<% content_for :page_title, @group.heading %>
+<% content_for :title, @group.heading %>
+<% content_for :context, @collection.title %>
+<% content_for :title_margin_bottom, 4 %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body">
+      <%= view_on_website_link_for @collection, url: { draft: true }, class: "govuk-link", target: "blank" %>
+    </p>
+
+    <%= render "components/secondary_navigation", {
+      aria_label: "Document collection group navigation tabs",
+      items: secondary_navigation_tabs_items(@group, request.path)
+    } %>
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-three-quarters">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "Documents",
+          margin_bottom: 6,
+        } %>
+      </div>
+
+      <div class="govuk-grid-column-one-quarter">
+        <p class="govuk-body govuk-!-text-align-right">
+          <%= link_to "Add document", "#", class: "govuk-link" %>
+        </p>
+      </div>
+    </div>
+
+    <% if @group.memberships.present? %>
+      <div class="govuk-table--with-actions app-view-document-collection-group-memberships-index__table">
+        <%= render "govuk_publishing_components/components/table", {
+          rows: @group.memberships.map do |membership|
+            title =  membership.document.latest_edition.title
+            [
+              {
+                text: title,
+              },
+              {
+                text: link_to(sanitize("View #{tag.span(title, class: "govuk-visually-hidden")}"), membership.document.latest_edition.public_url, class: "govuk-link") +
```

Comment:
Minor thing but I think elsewhere the space between "View"/"Remove" links and the visually hidden text has been part of the hidden text. Otherwise the link underline spans the text and the space making it look slightly odd.

Reply:
Hi @davidtrussler - I don't see where else it has the space being part of the hidden text, do you have an example? 
I searched for `#{tag.span(` and the results all have the space outside the hidden span, with the one exception of World Locations.

Reply:
@minhngocd Oh yes, you're right. In fact I see now that the `govuk-visually-hidden` CSS class actually adds a space before that text so really we should delete the space here and it will be taken care of that way. But since we haven't done that anywhere else (apart from the one you mention) maybe it doesn't really matter. Does look a bit wrong to me though.

Reply:
@davidtrussler oh dear 🙈 i will add a ticket to the tech debt column

Reply:
Yes, good idea, we should make it all consistent but in the meantime it's not exactly a show-stopper! Might be worth a bit more thought as well because I have this idea (that might well be nonsense) that screen-readers get thrown when a mid-sentence word starts in uppercase, as it does here.